hadolint/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM ubuntu
+MAINTAINER someone@example.com
```

Hadolint (2.12.0)

[error] MAINTAINER is deprecated

(DL4000)